how-to-guides/switch-data-training.ipynb

```diff
@@ -0,0 +1,106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# How to switch data provider during training"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "User can easily switch data provider during the training using\n",
+    "[`set_data()`](https://pytorch.org/ignite/v0.4.5/generated/ignite.engine.engine.Engine.html#ignite.engine.engine.Engine.set_data). See an example in the\n",
+    "documentation of the method."
```

Comment:
> See an example in the documentation of the method.

Why we need to send user somewhere else. Let's make a self-contained how-to guide